Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -87,11 +93,43 @@ - (OCTCall *)callToChat:(OCTChat *)chat enableAudio:(BOOL)enableAudio enableVide
     return nil;
 }
 
+- (BOOL)enable:(BOOL)enable videoSendingForCall:(OCTCall *)call error:(NSError **)error
+{
+    BOOL success = [self setVideoBitrate:kDefaultVideoBitRate forCall:call error:error];
+
+    if (! success) {
+        return NO;
+    }
+
+    if (enable && (call.pausedStatus == OCTCallPausedStatusNone)) {
+        OCTFriend *friend = [call.chat.friends firstObject];
+        self.videoEngine.friendNumber = friend.friendNumber;
+        [self.videoEngine startSendingVideo];
+    }
+    else {
+        [self.videoEngine stopSendingVideo];
+    }
+
+    OCTRealmManager *manager = [self.dataSource managerGetRealmManager];
+    [manager updateObject:call withBlock:^(OCTCall *callToUpdate) {
+        call.videoIsEnabled = enable;
+    }];
+
+    return success;
+}
+
 - (BOOL)answerCall:(OCTCall *)call enableAudio:(BOOL)enableAudio enableVideo:(BOOL)enableVideo error:(NSError **)error
 {
     OCTToxAVAudioBitRate audioBitRate = (enableAudio) ? kDefaultAudioBitRate : OCTToxAVAudioBitRateDisabled;
     OCTToxAVVideoBitRate videoBitRate = (enableVideo) ? kDefaultVideoBitRate : kOCTToxAVVideoBitRateDisable;
```

Comment:
Once I figure out good video bitrates to use, I'll create the enums. I don't even know if 400 is a good default number? Any ideas?